src/views/product-downloads-view/boundary/components/install-callout/install-callout.module.css

```diff
@@ -32,3 +28,7 @@
 		grid-template-columns: 16px 1fr 90px;
 	}
 }
+
+.lastCard {
+	margin-top: 32px;
+}
```

Comment:
Looks like this was moved to `src/views/product-downloads-view/boundary/download.module.css`, is it still needed here?

Reply:
No, good catch!